execution-engine/src/fs.rs

```diff
@@ -644,6 +644,17 @@ impl FileSystem {
     /// A rust-style base implementation for `fd_write`. It directly calls `fd_pwrite` with the
     /// current `offset` of Fd `fd` and then calls `fd_seek`.
     pub(crate) fn fd_write(&mut self, fd: Fd, buf: &[u8]) -> FileSystemResult<Size> {
+        // Redirect writes to fd(0) and fd(1) to the host's stdout and stderr respectively
+        if fd.0 == 0 || fd.0 == 1 {
+            let s = std::str::from_utf8(&buf).expect("Found invalid UTF-8");
+            print!("{}", s);
```

Comment:
Somewhere, there is a debug printing function that will only print to `stdout` on the host if the policy `debug=true` flag is set.  That should be used here, rather than a `print!()`.

Reply:
Are you referring to the profile option (https://doc.rust-lang.org/cargo/reference/profiles.html), or a Veracruz policy?

Reply:
There is a debug flag in the Veracruz runtime and a related `debug` function but has been exposed to the execution engine. Also, note that some TEE might simply do not support to call `print!` or the `print!` does pass information cross the TEE boundary.